### PR TITLE
Fix: makes extra functions from the trace module private

### PR DESCRIPTION
**Pull request by Saksham-13**
Removes the functions not mentioned in issue #134 from the __all__ variable to make them private.

### CODE REVIEW

**Comment by nabobalis** [comment]
You will need to update the test files as they are now failing.

File: changelog/136.removal.rst
Comment (nabobalis) [inline changelog/136.removal.rst:1]: The filename should `.breaking.`

File: changelog/136.removal.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Removes the functions
+1. curvature_radius
+2. erase_loop_in_image
+3. initial_direction_finding
+4. loop_add
+from the __all__ variable in `sunkit_image.trace` to make them private.
```
Comment (nabobalis) [inline changelog/136.removal.rst:6]:
```suggestion
Made the following functions in `sunkit_image.trace` private:

1. ``curvature_radius`` (renamed to ``_curvature_radius``)
2. ``erase_loop_in_image`` (renamed to ``_erase_loop_in_image``)
3. ``initial_direction_finding`` (renamed to ``_initial_direction_finding``)
4. ``loop_add`` (renamed to ``_loop_add``)

These were never intended to be used by users but for the user-facing functions.
```

**Comment by Saksham-13** [comment]
I think this PR is ready to be merged now

**Review by nabobalis — APPROVED** [review]

**Comment by nabobalis** [comment]
Thanks for the PR @Saksham-13

**Pull request merged**